README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
+  * Unique Repo
+    * **[Proposal]** ROS Maritime Robotics Drivers Index
+    * **[Proposal]** Sonar and radar drivers
+  * References: 
+    *  [ROS Drivers](https://github.com/ros-drivers)
+
+* Documentation
+  * Description: Documentation and index for maritime robotics packages and organizations.
+  * Motive: There are already many organizations and repos for maritime robotics. It is important to map what it is already developed in the area as well as make tutorials for newcomers or people that want to contribute with maritime robotics development.
+  * Repositories
+    * **[Proposal]** ROS Maritime Robotics Documentation Index
+    * **[Proposal]** ROS Maritime Robotics Tutorials
+  * References: 
+    * [COLA2](https://iquarobotics.com/cola2): ROS and Gazebo
+    * [VRX](https://github.com/osrf/vrx): ROS and Gazebo
+    * [UUV Simulator](https://uuvsimulator.github.io/): ROS and Gazebo
+    * [Plankton](https://github.com/Liquid-ai/Plankton): ROS2 and Gazebo
+    * [Ignition native underwater vehicles](https://ignitionrobotics.org/api/gazebo/5.0/underwater_vehicles.html): Ignition 
+    * [MBARI Vehicle Simulated in Ignition](https://github.com/osrf/lrauv/): Ignition 
+    * [Project Dave](https://github.com/Field-Robotics-Lab/dave/wiki): ROS and Gazebo
+    * [DS SIM](https://bitbucket.org/whoidsl/ds_sim/src/master/): ROS and Gazebo
+    * [UW APL Hydrographic Messages](https://github.com/apl-ocean-engineering/hydrographic_msgs): ROS
+  
+* Dynamic Model
+  * Description: Packages related to estimation and iteration of maritime robots models.
+  * Motive: Maritime robots dynamical models have to take into account buoayancy, hydrodynamics... Therefore some packages regarding math models are needed. 
+  * Repositories
+    * **[Proposal]** Maritime vehicle model estimation based on URDF
+      * Use vehicle mesh and parameters to estimate the Fossen equation of motion parameters
+    * **[Proposal]** Maritime vehicle estimation based experiments
+      * Use control, pose and velocity bags and URDF to estimate the Fossen equation of motion parameters
+    * **[Proposal]** Maritime vehicle model
+      * Use vehicle equations of motion to predict robot next position and velocity. This can be used with localization and control algorithms.
+  * References: 
+    *  Python lib that estimates added mass: [Capytaine](https://github.com/mancellin/capytaine)
+    *  [OpenFoam](https://www.openfoam.com/)
+
+* Guidance, Navigation and Control (GNC)  
+  * Description: Packages for gnc of maritime robots.
+  * Motive: There are some control and tasks specifics to maritime robots, such as thruster control allocation, surveilance, vessels handling of tides...
+  * Repositories
+    * **[Proposal]** Thruster manager
+  * References: 
+    *  [Plankton thruster manager](https://github.com/Liquid-ai/Plankton/blob/master/uuv_control/uuv_thruster_manager/src/uuv_thrusters/thruster_manager.py)
+ 
+* Localization
```

Comment:
With regards to this section, this is a pretty major task. There are multiple possible ways in which a vehicle shall localize. Are we focused on the underwater (harder) or surface use case (easier). If so what sensors? Also what are available benchmarks and datasets.